src/components/data/DataTable/TableRowExpandable.tsx

```diff
@@ -16,20 +17,23 @@ import PropTypes from 'prop-types';
  * </DataTable.RowExpandable>
  * ```
  */
-export default function TableRowExpandable() {}
-
-TableRowExpandable.propTypes = {
+export interface TableRowExpandableProps {
     /**
      * expanded row content
      */
-    children: PropTypes.node.isRequired,
+    children: ReactNode[];
 
     /**
      * if true, then expandable part of the row will be shown
      */
-    expanded: PropTypes.bool
+    expanded: boolean;
+}
+const TableRowExpandable: FunctionComponent<TableRowExpandableProps> = () => {
```

Comment:
Shouldn't we have
```suggestion
const TableRowExpandable: FunctionComponent<TableRowExpandableProps> = ({ expanded = false }) => {
```
here? 🤔

Reply:
in this case, this is not the case, since this render function is never actually run.

In general, i tried to not touch this part very much, since its a bit tricky (this component just acts as a vessel for children, and is unpacked and not rendered when its used properly (inside as table))